asm/patches/arc.asm

```diff
@@ -39,3 +39,18 @@ b 0x7100659ae0
 ; Use custom bzs.arc when trying to load vanilla bzs
 .offset 0x7100deb9a4
 bl 0x7100659ae8
+
+
+; Load stage arcs from romfs/ModReplace where possible
+; Unfortunately necessary jumptable usage ;-;
+.offset 0x7100b8c3e4
+bl 0x7100659af0
+
+; Load general arcs from romfs/ModReplace where possible
+.offset 0x7100deb2cc
+mov w8, #96 // replaced instructions / function setup
```

Comment:
Is using `//` okay for comments instead of `;`?

Reply:
I guess they must be okay since the assembler didn't complain about them. I'll change them to `;` tomorrow for consistency tho :p